go-controller/pkg/ovn/policy.go

```diff
@@ -766,6 +768,24 @@ func (oc *Controller) destroyNamespacePolicy(np *namespacePolicy) {
 	}
 }
 
+// handlePeerServiceSelectorAddUpdate adds the VIP of a service that selects
+// pods that are selected by the Network Policy
+func (oc *Controller) handlePeerServiceSelectorAddUpdate(gp *gressPolicy, obj interface{}) {
+	service := obj.(*kapi.Service)
+	if err := gp.addPeerSvcVip(service); err != nil {
+		klog.Errorf(err.Error())
+	}
+}
```

Comment:
Is there no way that a service can have a VIP removed during an update operation?